### PR TITLE
Adopt changes from the BME280 library

### PR DESCRIPTION
**Pull request by jraber**
Adopt changes from the BME280 library to enable Normal mode, IIR Filters, & Overscan

I applied the changes that were applied to the BME280 library to the BMP280.

### CODE REVIEW

**Comment by jraber** [comment]
ping.

Is there something I can do to help get this reviewed?

This is basically just the same code that was already accepted in the BME280 library, minus the humidity property (since the BMP280 doesn't have that).

Ref: https://github.com/adafruit/Adafruit_CircuitPython_BME280/pull/22

**Comment by ladyada** [comment]
@siddacious wanna take a look at this PR & test it?

**Comment by siddacious** [comment]
Sure

**Comment by siddacious** [comment]
@ladyada I just checked and I don't have a BME280 but I'll get one ordered

**Comment by makermelissa** [comment]
@siddacious, this PR is for the BMP280.

**Comment by siddacious** [comment]
@makermelissa thanks for the catch. I also do not have one of those

**Comment by makermelissa** [comment]
I can take a look then. I have both.

**Comment by siddacious** [comment]
@makermelissa ok, sounds good

File: examples/bmp280_normal_mode.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import adafruit_bmp280
+
+# Create library object using our Bus I2C port
+i2c = busio.I2C(board.SCL, board.SDA)
+bmp280 = adafruit_bmp280.Adafruit_BMP280_I2C(i2c)
+
+# OR create library object using our Bus SPI port
+#spi = busio.SPI(board.SCK, board.MOSI, board.MISO)
+#bme_cs = digitalio.DigitalInOut(board.D10)
+#bme280 = adafruit_bme280.Adafruit_BME280_SPI(spi, bme_cs)
```
Comment (makermelissa) [inline examples/bmp280_normal_mode.py]: Looks like this was left as bme280. Just need to change it to:
```
bmp_cs = digitalio.DigitalInOut(board.D10)
bmp280 = adafruit_bmp280.Adafruit_BMP280_SPI(spi, bmp_cs)
```

**Review by makermelissa — CHANGES_REQUESTED** [review]
Tested both examples on SPI and I2C. Just one small change.

File: examples/bmp280_normal_mode.py (hunk 1 above)
Comment (jraber) [reply examples/bmp280_normal_mode.py]: Good catch.  Fixed that with commit b9e41db49

**Review by makermelissa — APPROVED** [review]

**Comment by makermelissa** [comment]
Looks good.

**Comment by ladyada** [comment]
hiya this broke trinket m0 and other non-express builds. do we need to use `frozenset` or can we use something else?
https://forums.adafruit.com/viewtopic.php?f=52&t=151982

**Comment by makermelissa** [comment]
I can test this tonight and see if I can figure how to get it working with the trinket.

**Comment by makermelissa** [comment]
Removing `frozenset` and using a normal list fixed the issue. I'll submit a PR.

**Pull request merged**